generic3g/actions/MinAccumulator.F90

```diff
@@ -7,10 +7,9 @@ module mapl3g_MinAccumulator
    use ESMF
    implicit none
    private
-   public :: AccumulatorAction
+   public :: MinAccumulator
```

Comment:
I really prefer all of the subclasses to end in "Action".    Am open to reasons why that is not appropriate here.

Reply:
Done